parsimonious/grammar.py

```diff
@@ -226,8 +227,8 @@ def _expressions_from_rules(self, rule_syntax, custom_rules):
     literal = spaceless_literal _
 
     # So you can't spell a regex like `~"..." ilm`:
-    spaceless_literal = ~"u?r?\"[^\"\\\\]*(?:\\\\.[^\"\\\\]*)*\""is /
-                        ~"u?r?'[^'\\\\]*(?:\\\\.[^'\\\\]*)*'"is
+    spaceless_literal = ~"u?r?b?\"[^\"\\\\]*(?:\\\\.[^\"\\\\]*)*\""is /
+                        ~"u?r?b?'[^'\\\\]*(?:\\\\.[^'\\\\]*)*'"is
```

Comment:
This change adds support for parsing bytestrings by allowing you to specify bytestring literals inside of grammars. Amazingly, this is literally all that is required to add that functionality.